src/app.tsx

```diff
@@ -127,47 +128,63 @@ export class AppState extends EventEmitter<AppStateEvents> {
         if (this.#initPromise)
             return this.#initPromise;
 
+        const init_connection = async (connectionName: ConnectionName) => {
+            try {
+                await getLibvirtVersion({ connectionName });
+                const promises = await getApiData({ connectionName });
+                const errorMsgs = promises
+                        .filter(promise => promise.status === 'rejected')
+                        .map(promise => promise.reason.message);
+                if (errorMsgs.length > 0) {
+                    addNotification({
+                        text: _("Failed to fetch some resources"),
+                        detail: errorMsgs.join(', ')
+                    });
+                }
+                // Get the node devices in the background since
+                // they are expensive to get and not important for
+                // displaying VMs.
+                nodeDeviceGetAll({ connectionName }).catch(exc => {
+                    addNotification({
+                        text: "Failed to retrieve node devices",
+                        detail: String(exc),
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2466-9900a690-20251229-144551-fedora-43-devel/Coverage/lcov/github-pr.diff.gcov.html)